src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -20,13 +21,16 @@ namespace Microsoft.Azure.WebJobs.Extensions.ApiHub
     [Singleton(Mode = SingletonMode.Listener)]
     internal class ApiHubListener : IListener
     {
+        private const int MaxFunctionExecutionRetryCount = 5;
         private const string HostContainerName = "azure-webjobs-hosts";
+        private const string PoisonQueueName = "azure-webjobs-apihub-poison";
```

Comment:
Thinking you should follow the naming scheme for blob trigger https://azure.microsoft.com/en-us/documentation/articles/websites-dotnet-webjobs-sdk-storage-blobs-how-to/

E.g. **webjobs-apihubtrigger-poison**